File: Assignment 2/if else/Vote.cpp
#include<iostream>
using namespace std;
class Eligibility
{

    int age;
    char yes,no;
    public:void vote()

    {
        cout<<"Enter the age of the person:\n";
        cin>>age;
        if(age>=18)
        {
            cout<<"The person is eligible to vote";
            cin>>yes;
        }
        else
        {
            cout<<"The person is not eligible to vote";
            cin>>no;
        }
    }
};
int main()
{
    Eligibility obj;
    obj.vote();
}

File: Java assignment(1-2)/java Assignment 2/Vote.java
import java.util.Scanner;
class Vote
{
int num1;
void getData()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter the age : ");
num1=sc.nextInt();
if(num1>=18)
{
System.out.println("The person is eligible to vote");
}
else
{
System.out.println("The person is not eligible to vote");
}
}
public static void main(String...args)
{
Vote obj=new Vote();
obj.getData();
}
}
